app/javascript/src/Stats/inlinechart/types/index.jsx

```diff
@@ -0,0 +1,28 @@
+// @flow
```

Comment:
Keep the types in the same file. specially if they are gonna be use just in one place.